cdap-master/src/main/java/co/cask/cdap/data/tools/ReplicationStatusTool.java

```diff
@@ -79,17 +80,31 @@
   private static String outputStatusFileName = null;
   private static String dirsFileName = null;
   private static boolean debugDump = false;
-  private static String delimiter = "#";
-  private static String checksumsEnd = "EndofChecksums";
-  private static String tablesEnd = "EndofTables";
+
+  private static final Gson GSON = new Gson();
 
   // Default paths to be pulled from configuration
   private static final String[] ALL_CHECKSUM_PATHS_DEFAULT = {
-    "data.tx.snapshot.dir" // this is only defined at tephra/TxConstants.java
+    CFG_TX_SNAPSHOT_DIR // this is defined at tephra/TxConstants.java
```

Comment:
to use the same style as elsewhere, use TxConstants.Manager.CFG_TX_SNAPSHOT_DIR here, and import org.apache.tephra.TxConstants